install/kubernetes/helm/istio/charts/sidecarInjectorWebhook/templates/deployment.yaml

```diff
@@ -49,6 +49,7 @@ spec:
             - --meshConfig=/etc/istio/config/mesh
             - --healthCheckInterval=2s
             - --healthCheckFile=/health
+            - --enableSidecarInjectorToManageWebhookConfig={{ .Values.enableSidecarInjectorToManageWebhookConfig }}
```

Comment:
Can we set a default value here by adding `| default <true_or_false>`?

Reply:
The default value has been set in install/kubernetes/helm/istio/charts/sidecarInjectorWebhook/values.yaml. Moreover, if changing the line into " - --enableSidecarInjectorToManageWebhookConfig={{ .Values.enableSidecarInjectorToManageWebhookConfig | default true }}", the helm set command does not set the value of enableSidecarInjectorToManageWebhookConfig based on the value in the helm set command, e.g., the command "helm template --name=istio --namespace=istio-system --set sidecarInjectorWebhook.enableSidecarInjectorToManageWebhookConfig=false install/kubernetes/helm/istio > istio-test.yaml" will not set enableSidecarInjectorToManageWebhookConfig to false.

Reply:
Why not? 
I did't have a chance to try `helm template --name=istio --namespace=istio-system --set sidecarInjectorWebhook.enableSidecarInjectorToManageWebhookConfig=false install/kubernetes/helm/istio`
I thought the parameter in `--set` in `helm install` has the highest priority.

Reply:
According to https://github.com/helm/helm/blob/master/docs/chart_template_guide/values_files.md: "--set has a higher precedence than the default values.yaml file". But I am unable to find a helm document describing the precedence relationship between the helm default function and the helm set command. If you can find such document, please let us know.